changelogs/fragments/79839-lookup-option-name.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - templates - Fixed `TypeError` when a lookup plugin has an option called `name`.
```

Comment:
FYI this is not markdown but rst

Reply:
would you like me to change it?

Reply:
It would be a welcome improvement if you have a minute. Using double backticks should be enough.

Reply:
I've opened up #80065 with the changes.